dataans/src-tauri/src/dataans/command/import/json.rs

```diff
@@ -0,0 +1,69 @@
+use std::fs::File;
+use std::path::Path;
+use std::sync::Arc;
+
+use common::export::{Schema, SchemaV1};
+use futures::future::join_all;
+use serde_json;
+
+use crate::dataans::db::Db;
+use crate::dataans::service::note::NoteService;
+use crate::dataans::service::space::SpaceService;
+use crate::dataans::DataansError;
+
+pub async fn import_v1<D: Db>(
+    schema_v1: SchemaV1,
+    space_service: &Arc<SpaceService<D>>,
+    note_service: &Arc<NoteService<D>>,
```

Comment:
Do you really need to use `Arc` here? Did it fail to compile without the `Arc` wrapper?